play-ws-standalone/src/main/java/play/libs/ws/StandaloneWSResponse.java

```diff
@@ -122,6 +122,15 @@
      *
      * Note that this is only usable with a streaming request:
      *
+     * <pre>
+     * {@code
+     * wsClient.url("https://playframework.com")
+     *         .stream() // this returns a CompletionStage<StandaloneWSResponse>
+     *         .thenApply(StandaloneWSResponse::getBodyAsSource);
+     * }
+     * </pre>
      */
-    Source<ByteString, ?> getBodyAsSource();
+    default Source<ByteString, ?> getBodyAsSource() {
+        return Source.single(getBodyAsBytes());
```

Comment:
OK so the default implementation of a `Source` calls `getBodyAsBytes()`, which loads the entire response into memory. Do we override this somewhere to build a source from something streaming, like AHC's `response.getResponseBodyAsStream()`?